app/services/protocol-links.ts

```diff
@@ -69,4 +71,12 @@ export class ProtocolLinksService extends Service {
       });
     }
   }
+
+  @protocolHandler('paypalauth')
+  private updateUserBillingInfo(info: IProtocolLinkInfo) {
+    // 1. navigate to store tab
+    // 2. trigger callback to store webview to continue checkout
+    this.navigationService.navigate('PlatformAppStore');
```

Comment:
If the user has navigated to a different page, the webview will have been deloaded, and the binding on paypal success will have been lost.  This effectively makes this a failure state and I'm not sure if it would be possible for us to easily recover from that.